### PR TITLE
Add image previews to README.md

### PR DESCRIPTION
**Pull request by DennisRutjes**
The README.md document has been updated with image previews that demonstrate the expected result of certain code examples. These changes provide a visual reference to help users understand the output or effect of the provided code. A new SVG file 'preview3.svg' has also been added to the images directory for this purpose.

### CODE REVIEW

**Comment by DennisRutjes** [comment]
In the issues section User asked a for preview images, so i added them

**Comment by kt3k** [comment]
`images/preview2.svg`, `images/preview3.svg`, and `images/preview4.svg` don't seem corresponding to the examples above them.

I don't think it's good idea to add image previews for each code example, as all code examples show the same charts.

**Comment by emilkrebs** [comment]
Looks great to me.

File: README.md [hunk 1]
```diff
@@ -61,6 +61,10 @@ export default function Home() {
 }
 ```
 
+### [SSR] Inline chart example preview
+
+![](./images/preview1.svg)
```
Comment (emilkrebs) [inline README.md:66]: very nice, didn't know you could use repository-specific paths. Will definitely keep this in mind. Thanks!

**Review by emilkrebs — APPROVED** [review]
Looks great!

**Comment by emilkrebs** [comment]
> `images/preview2.svg`, `images/preview3.svg`, and `images/preview4.svg` don't seem corresponding to the examples above them.
> 
> I don't think it's good idea to add image previews for each code example, as all code examples show the same charts.

For myself, it would be helpful knowing what the charts would look like. It makes it easier to consider which one to use.

**Pull request closed without merging**